syntax = "proto3";

package userservice;

service UserService {
  rpc CreateUser (CreateUserRequest) returns (UserResponse) {}
  rpc GetUser (GetUserRequest) returns (UserResponse) {}
}

message CreateUserRequest {
  string name = 1;
  string password = 2;
}

message GetUserRequest {
  string name = 1;
  string password = 2;
}

message UserResponse {
  string id = 1;
  string name = 2;
  string token = 3;
}
